app/views/posts/_feed.html.erb

```diff
@@ -0,0 +1,55 @@
+<div class="column is-half">
+    <% if user_signed_in? %>
+    <article class="media box">
+        <figure class="media-left">
+            <p class="image is-64x64">
+						<%= gravatar_image_tag(current_user.email, size: 64, alt: current_user.name) %>
+            </p>
+        </figure>
+        <div class="media-content">
+            <%= render 'posts/form' %>
+        </div>
+    </article>
+    <%end%>
+
+
+<% @posts.each do |post| %>
+   <div class="box">
+    <article class="media ">
+    <% if user_signed_in? %>
+       <div class="image is-64x64">
+            <figure class="media-left">
+						<%= gravatar_image_tag(post.user.email, size: 64, alt: post.user.name) %>
+            </figure>
+        </div>
+    <%end%>
+        <div class="media-content">
+            <div class="content">
+                <% if user_signed_in? %>
+                    <strong><%= post.user.name %></strong><br/>
```

Comment:
Move this logic to a helper. As highlighted in the last review. There should be absolutely no logic in view.